src/cheri-pte-ext.adoc

```diff
@@ -141,11 +141,18 @@ bit value of the capability read. This will introduce additional traps during re
 |===
 |PTE.CW |PTE.CRG |Store/AMO
 | 0     | 0      | Page fault if stored tag is set
-| 0     | 1      | Page fault if stored tag is set, or hardware CW and CRG update^1^
+| 0     | 1      | Page fault if stored tag is set, or hardware CW and CRG update^2^
 | 1     | X      | Normal operation
 |===
 
-^1^ The choice here follows the pattern of whether to implement the _Svade_ extension
+^1^ The choice here is whether to take a data dependent exceptions on loads or atomic operations.
+ If the exception is taken then the software will sweep the page and update the generation
+ (CRG). If the exception is taken without checking the tag, then the sweep will happen sooner.
+ The choice on whether to check the tag or not can be static or dynamic. It is also permitted to
```

Comment:
Do you mean that the data-dependent check can be turned on/off at run-time, for example, by writing a CSR?

Reply:
it can vary as the implementor sees fit